README.md

```diff
@@ -476,3 +476,38 @@ The following flags are available for `patch subscription`:
 ##### Examples
 
 * Patch a subscription by its ID and change the status to 'Reserved' (dry run) `echo '{ "status": "Reserved" }' | ocm support patch subscriptions [subID]`
+
+### Sync cloud resources
+
+OCM Support CLI can be used to sync cloud resources and generate quota rules in AMS.
+
+This command involves:
+1. cloning uhc-account-manager repo
+2. replacing quota-cloud-resources.csv
+3. generating quota rules by executing `make generate-quota`
+4. pushing to a remote branch in AMS so an MR can be opened (note: it does not currently open the MR for you, but does provide the gitlab link to open the MR) 
+
+Note that the command does NOT support the following:
+- Removing cloud resources
+- Adding incorrect cloud resources for which quota rules cannot be generated
```

Comment:
what do you mean by this?

Reply:
Incorrect cloud resources are the ones that are not valid resource names